src/graphql/error-map.ts

```diff
@@ -271,6 +271,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "Invalid walletId for account."
       return new ValidationInternalError({ message, logger: baseLogger })
 
+    case "SecretDoesNotMatchAnyExistingHodlInvoiceError":
+      message = `Unknown error occurred (code: ${error.name})`
```

Comment:
This message is a duplicate. The case can be added to the unhandled list below Ln 280

Reply:
yup thanks